website/blog/2021-02-08-webdriverio-v7-released.md

```diff
@@ -0,0 +1,182 @@
+
+### How to Update?
+
+The are two significant changes how this TypeScript rewrite will impact you. While in v6 you would set `@wdio/sync` in your types of your `tsconfig.json` this will now change to be `webdriverio/sync`. Similar for running WebdriverIO asynchronous. Instead of just defining `webdriverio` in your types you now need to set `webdriverio/async`:
+
+```git
+// tsconfig.json
+"types": [
+  "node",
+-  "@wdio/sync",
++  "webdriverio/sync",
+  "@wdio/mocha-framework"
+],
+```
+
+or
+
+```git
+// tsconfig.json
+"types": [
+  "node",
+-  "webdriverio",
++  "webdriverio/async",
+  "@wdio/mocha-framework"
+],
+```
+
+Lastly if you define custom commands you need to provide their types slightly different now:
+
+```ts
+// define custom commands in v6
+declare namespace WebdriverIO {
+    // adding command to `browser`
+    interface Browser {
+        browserCustomCommand: (arg: number) => void
+    }
+}
+```
+this now has to be:
+```ts
+declare global {
+    namespace WebdriverIO {
+        interface Browser {
+            browserCustomCommand: (arg: number) => void
+        }
+    }
+}
+```
+
+You can read more about WebdriverIO TypeScript integration in our [docs](/docs/typescript).
+
+## Cucumber v7 Update
+
+The folks working on Cucumber have done a tremendous job moving their code base to TypeScript to which has made our lives tremendously easier. The new Cucumber integration required us to update the parameters within our [Cucumber hooks](/docs/options#beforefeature).
+
+### How to Update?
+
+If you have been using Cucumber all you need to do to update to v7 is to update your Cucumber imports to their new package:
+
+```git
+- const { Given, When, Then } = require('cucumber')
++ const { Given, When, Then } = require('@cucumber/cucumber')
+```
+
+## Improved Google Lighthouse Integration
+
+Since v6 WebdriverIO can run on the [WebDriver protocol](https://w3c.github.io/webdriver/) for true cross browser automation but also automate specific browser using browser APIs such as [Chrome DevTools](https://chromedevtools.github.io/devtools-protocol/). This allows for interesting integrations into tools that allow broader testing capabilities such as [Google Lighthouse](https://developers.google.com/web/tools/lighthouse). With the `@wdio/devtools-service` WebdriverIO users were able to access these capabilities for a while now using Google Ligthouse to run [performance tests](https://www.youtube.com/watch?v=Al7zlLdd_es). In this released with have also updated Google Lighthouse to the latest version to enable new performance metrics such as [Cumulative Layout Shifts](https://web.dev/cls/) or [First Input Delay](https://web.dev/fid/).
```

Comment:
'In this released with have also updated' -> 'In this release, we have also updated'